### ISSUE TITLE
Add the ability to have multiple download lists

### ISSUE THREAD

**Issue by smortex** [puppetlabs/forge_issues#11]
## Use Case
As a user, I may manage multiple Puppet infrastructures and contribute to multiple Bolt projects.  Each one has it's Puppetfile that I want to be up-to-date.

## Describe the Solution You Would Like
I would like to have the ability to have multiple download lists, for which I provide a unique name and where I can upload a Puppetfile.  As uploading a Puppetfile clears the list, this would only affect the current list and the other ones would be kept unchanged.

## Describe Alternatives You've Considered
n/a

## Additional Context
Basically it is about changing a 1-1 relationship between *users* and *download lists* to a 1-*.

With organization/delegation support (see #10), a download list could live at the user or organization level

**Comment by skarwaperforce** [comment]
Hi @smortex thanks for the feedback, this is an enhancement we have in the backlog, adding your issue to the request. Thanks!
